source/support/communicating-with-users/common-responses.html.md

```diff
@@ -294,6 +294,29 @@ If a site, service or product (inside or outside of government) is found to be i
 
 Redirect: govuk-enquiries@digital.cabinet-office.gov.uk and/or contact the Policy & Strategy team on Slack for guidance (#govuk-policy-engage)
 
+### Access and usage of new coat of arms, crest,  department logo, cypher etc.
+
+> We only have the image of the coat of arms for use on the  GOV.UK footer. If you need access or usage information to other assets, you will need to email the Government Communication Service (GCS) branding@cabinetoffice.gov.uk.
+
+### Deadline for updating to new coat of arms
+
+> There is no specific deadline to update the coat of arms in the footer of your service. 
+> 
+> You should update at your earliest convenience. This will ensure consistency across all GOV.UK pages, building trust and reducing risk of fraud for citizens using your service.
+
+### File and format of new coat of arms
+
+> You can extract the image from GOV.UK Frontend v4.9.0 or v5.7.0. We provide the image as a SVG file. You can convert this to the format you need.
```

Comment:
As suggested by @owenatgov on the draft, I think we can hold hands a little bit there to avoid further back and forth on how to extract the image and where from. I'd propose the following (we can update the link once we merge https://github.com/alphagov/govuk-frontend/pull/5376/): 

```suggestion
> You can <a href="https://raw.githubusercontent.com/alphagov/govuk-frontend/9b57da6f0b8194473d67288666bc7d100339f289/packages/govuk-frontend/src/govuk/assets/images/govuk-crest.svg">download a SVG version of the coat of arms</a> from GOV.UK Frontend's repository. 
> 
> If you require another format, like PNG, many graphic editors will allow you to convert it to suit your needs. We unfortunately can't provide all combinations of formats and image dimensions, so that part is up to you.
```